Addition of full file paths in security documentation (#8113)
opensearch-project/documentation-website: _security/configuration/yaml.md

## action_groups.yml

> This file contains any role mappings required for your security configuration. You can find the `role_mapping.yml` file in <OPENSEARCH_HOME>/config/opensearch-security/roles_mapping.yml.

Reviewer: ```suggestion
This file contains any role mappings required for your security configuration. You can find the `role_mapping.yml` file in `<OPENSEARCH_HOME>/config/opensearch-security/roles_mapping.yml`.
```

## internal_users.yml

> Typically found at `<OPENSEARCH_HOME>/config/opensearch-security/internal_users.yml`, this file contains any initial users that you want to add to the Security plugin's internal user database.

Reviewer: ```suggestion
This file contains any initial users that you want to add to the Security plugin's internal user database. You can this file in ``<OPENSEARCH_HOME>/config/opensearch-security/internal_users.yml`.
```

  description: "Demo snapshotrestore user"
```

## opensearch.yml

> In addition to many OpenSearch settings, the `opensearch.yml` file contains paths to TLS certificates and their attributes, such as distinguished names and trusted certificate authorities. You can this file in  `<OPENSEARCH_HOME>/config/`.

Reviewer: ```suggestion
In addition to many OpenSearch settings, the `opensearch.yml` file contains paths to TLS certificates and their attributes, such as distinguished names and trusted certificate authorities. You can find this file in  `<OPENSEARCH_HOME>/config/`.
```

```

## opensearch.yml

> Typically found at `<OPENSEARCH_HOME>/config/opensearch.yml`. In addition to many OpenSearch settings, this file contains paths to TLS certificates and their attributes, such as distinguished names and trusted certificate authorities.

Reviewer: ```suggestion
In addition to many OpenSearch settings, the `opensearch.yml` file contains paths to TLS certificates and their attributes, such as distinguished names and trusted certificate authorities. You can this file in  `<OPENSEARCH_HOME>/config/`.
```

```

## allowlist.yml

> Typically found at `<OPENSEARCH_HOME>/config/opensearch-security/allowlist.yml`, this file can be used to add any endpoints and HTTP requests to a list of allowed endpoints and requests. If enabled, all users except the super admin are allowed access to only the specified endpoints and HTTP requests, and all other HTTP requests associated with the endpoint are denied. For example, if GET `_cluster/settings` is added to the allow list, users cannot submit PUT requests to `_cluster/settings` to update cluster settings.

Reviewer: ```suggestion
This file can be used to add any endpoints and HTTP requests to a list of allowed endpoints and requests. If enabled, all users except the super admin are allowed access to only the specified endpoints and HTTP requests, and all other HTTP requests associated with the endpoint are denied. For example, if GET `_cluster/settings` is added to the allow list, users cannot submit PUT requests to `_cluster/settings` to update cluster settings. You can find the `allowlist.yml` file in `<OPENSEARCH_HOME>/config/opensearch-security/allowlist.yml`.
```

## roles.yml

> Typically found at `<OPENSEARCH_HOME>/config/opensearch-security/roles.yml`, this file contains any initial roles that you want to add to the Security plugin. Aside from some metadata, the default file is empty, because the Security plugin has a number of static roles that it adds automatically.

Reviewer: ```suggestion
This file contains any initial roles that you want to add to the Security plugin. Aside from some metadata, the default file is empty, because the Security plugin has a number of static roles that it adds automatically. You can find the `roles.yml` file in `<OPENSEARCH_HOME>/config/opensearch-security/roles.yml`.
```

## roles_mapping.yml

> Typically found at `<OPENSEARCH_HOME>/config/opensearch-security/roles_mapping.yml`.

Reviewer: This file contains any role mappings required for your security configuration. You can find the `role_mapping.yml` file in `<OPENSEARCH_HOME>/config/opensearch-security/roles_mapping.yml`.
